app/Presenters/Admin/DigitalPublicationArticlePresenter.php

```diff
@@ -14,9 +13,7 @@ public function getCanonicalUrl()
 
     public function type()
     {
-        if ($this->entity->type) {
-            return DigitalPublicationArticle::$types[$this->entity->type];
-        }
+        return $this->entity->type->name;
```

Comment:
Do we no longer need a catch here if `type` is not set?

Reply:
I made `type` a `NOT NULL` field:
https://github.com/art-institute-of-chicago/artic.edu/pull/411/files#diff-668a90066b9b8d647af555cfabb20d4eed468aaf350e0c33bfbc115e7d5354b0R14

Do we need to allow for articles that don't have a type?

Reply:
No, you're implementation looks good!